apps/app-frontend/src/components/ui/Instance.vue

```diff
@@ -1,19 +1,26 @@
 <script setup>
-import { onUnmounted, ref, computed, onMounted } from 'vue'
-import { useRouter } from 'vue-router'
-import { SpinnerIcon, GameIcon, TimerIcon, StopCircleIcon, PlayIcon } from '@modrinth/assets'
-import { ButtonStyled, Avatar } from '@modrinth/ui'
-import { convertFileSrc } from '@tauri-apps/api/core'
-import { kill, run } from '@/helpers/profile'
-import { get_by_profile_path } from '@/helpers/process'
-import { process_listener } from '@/helpers/events'
-import { handleError } from '@/store/state.js'
-import { showProfileInFolder } from '@/helpers/utils.js'
-import { handleSevereError } from '@/store/error.js'
-import { trackEvent } from '@/helpers/analytics'
+import {computed, onMounted, onUnmounted, ref} from 'vue'
```

Comment:
seems like you may have a formatter that is reformatting all this stuff? hopefully running `pnpm fix` fixes this?

Reply:
I think my `pnpm fix` got overridden when I committed.